app/models/miq_event.rb

```diff
@@ -14,8 +14,9 @@ class MiqEvent < EventStream
     }
   }
 
-  SUPPORTED_POLICY_AND_ALERT_CLASSES = [Host, VmOrTemplate, Storage, EmsCluster, ResourcePool, MiqServer,
-                                        ExtManagementSystem, ContainerImage].freeze
+  SUPPORTED_POLICY_AND_ALERT_CLASSES = [Host, VmOrTemplate, Storage, EmsCluster, ResourcePool,
+                                        MiqServer, ExtManagementSystem,
+                                        ContainerReplicator, ContainerGroup, ContainerNode, ContainerImage].freeze
```

Comment:
@cben Is ContainerImage needed here? Seems there is no miq_event raised to ContainerImage.